.github/workflows/nightly.yaml

```
@@ -0,0 +1,31 @@
+name: Nightly Tests
+
+on:
+  schedule:
+    - cron: "0 0 * * *" # Runs every midnight
+  pull_request:
+    paths:
+      - .github/workflows/nightly.yaml
+
+permissions:
+  contents: read
+
+jobs:
+  integration-tests-terraform:
+    runs-on: self-hosted-linux-amd64-noble-large
+    steps:
+    - name: Setup operator environment
+      uses: charmed-kubernetes/actions-operator@main
+      with:
+        provider: lxd
+        channel: latest/stable
+        juju-channel: 3.6/stable
```

We can use the latest `3` version by tracking `3/stable`. Wdyt?
```suggestion
        juju-channel: 3/stable
```